src/component/registry/css-registry.ts

```diff
@@ -55,9 +55,8 @@ export class CssRegistry {
    * @param classNames the CSS class names to register
    * @return `true` if at least one class name from parameters has been added; `false` otherwise
    */
-  addClassNames(bpmnElementId: string, classNames: string[]): boolean {
-    return this.updateClassNames(bpmnElementId, classNames, (currentClassNames, className) => currentClassNames.add(className));
-  }
+  addClassNames = (bpmnElementId: string, classNames: string[]): boolean =>
```

Comment:
**praise:** Nice writing 🙂

Reply:
Well, my IntelliJ did the conversion and estlint did the rest 😸